.github/workflows/test-e2e.yml

```diff
@@ -138,6 +138,7 @@ jobs:
             "deployment-ns",
             "deployment-wide",
             "encryption-at-rest",
+            "encryption-at-rest-gcp",
```

Comment:
Why separate it from other EaR tests?

Reply:
Ok, separation was more due to flakiness than time though.

Reply:
done